dataland-frontend/src/components/general/TheHowItWorks.vue

```diff
@@ -0,0 +1,281 @@
+<template>
+  <section v-if="howItWorksSection" class="howitworks" role="region" aria-labelledby="howitworks-heading">
+    <div class="howitworks__wrapper">
+      <h2 id="howitworks-heading" class="howitworks__title">
+        {{ sectionText }}
+      </h2>
+      <div ref="slider" role="list" class="howitworks__slides" @pointerdown="dragStart" @touchstart="dragStart">
+        <div v-for="(slide, index) in slides" :key="index" role="listitem" class="howitworks__slide">
+          <h3 class="howitworks__slide-title">{{ slide.title }}</h3>
+          <p class="howitworks__slide-text">{{ slide.text }}</p>
+          <p class="howitworks__slide-index">0{{ index + 1 }}</p>
+        </div>
+      </div>
+      <div class="howitworks__arrows">
+        <button
+          @click="move(-1)"
+          aria-label="Previous slide"
+          class="howitworks__arrow howitworks__arrow--left"
+        ></button>
+        <button @click="move(1)" aria-label="Next slide" class="howitworks__arrow howitworks__arrow--right"></button>
```

Comment:
Closing "</button>" tags could be removed.

Reply:
What do you mean?
Attribute values should be enclosed in quotes for clarity and to conform to HTML standards.

Reply:
Oops sorry, seems I had a typo in that comment.

I meant the closing `</button>` tags are unnecessary for the arrow buttons.   Sorry for confusion.

Reply:
Ok sorry I did not know that.
To me it always seemed as if this was possible. One example from our code base (even though it is not a `<button>` element):

![image](https://github.com/d-fine/Dataland/assets/99255296/41b3851a-4e6c-4432-9f75-519c76478d6e)

Reply:
If you forget to close a <button> tag in HTML, the browser will try to auto-correct the mistake. This usually means the content following the unclosed <button> tag may be treated as part of the button until a closing tag is encountered or the end of the document is reached. This can lead to unexpected layout and behavior.

In Vue, the template compiler would also try to handle the unclosed tag but it's good practice to close all HTML tags properly to avoid any unforeseen issues.

Reply:
Using a self-closing `<i>` tag like in your example is generally not recommended for HTML5, as the `<i>` tag is not a void element and should have a closing tag. However, many browsers are forgiving and will try to correct the mistake.

In the context of Vue.js, if you're using single-file components, the template compiler can handle self-closing tags and will properly convert them into a pair of opening and closing tags during the build process. This means that `<i class="whatever" />` would essentially be treated as `<i class="whatever"></i>`.

Still, for the sake of readability and adherence to HTML5 standards, it's a good practice to use the opening and closing tags `<i class="whatever"></i>` instead.

Reply:
Ok cool. Thanks for the explanation. Learned something new 👍 .

Will close this conversation now.